app/views/events/index.html.erb

```diff
@@ -1,10 +1,41 @@
 <p style="color: green"><%= notice %></p>
 
 <h1>Events</h1>
-<%= link_to "New event", new_event_path %>
+<%#= link_to "New event", new_event_path %>
+<%= link_to "New event", new_event_path(start_date: [@calendar[:day].label, @calendar[:month].label(format: '%m-%Y')].join('-')) %>
```

Comment:
there should maybe be a better way to deduce current day/month/year, but it's fine

Reply:
IIUC you want to get the starting date of the current selected day (smallest time unit of your calendar), so you can use that to feed to the new form, right?

If that's the case it's easy to get: I just have to add an accessor for it, because I didn't think to use it to get data from.

So you could just do `start_date = @calendar[:day].from`.  Now there is no accessor so you would have to retrieve `@from`.

Or even better, I could create a generic method that gives you the `@calendar.showtime`... picking the beginning of the smallest time unit currently shown in your calendar.

Reply:
`@calendar.showtime` force-pushed to `dev`.

Reply:
Thank you 🙇‍♂️ will try it out now

Reply:
works great! no more questions!